pkg/eventpersister/persister/persister-dwh.go

```diff
@@ -0,0 +1,597 @@
+// Copyright 2022 The Bucketeer Authors.
```

Comment:
Because the DWH service uses a different container image, it will be better if we add the server to a separate package as you did in the `cmd/eventpersister-dwh` to keep consistency and easier understanding.

E.g.

```go 
pkg/eventpersister-dwh
```